wemake_python_styleguide/violations/base.py

```diff
@@ -99,11 +98,28 @@ class BaseViolation(object, metaclass=abc.ABCMeta):
     previous_codes: ClassVar[Set[int]]
     deprecated: ClassVar[bool] = False
 
+    # assigned in __init_subclass__
+    full_code: ClassVar[str]
+    summary: ClassVar[str]
+
     # We use this code to show base metrics and thresholds mostly:
     postfix_template: ClassVar[ViolationPostfixes] = (
         ViolationPostfixes.bigger_than
     )
 
+    def __init_subclass__(cls) -> None:
```

Comment:
This causes `WPS338` error. Considering that #2411 is in place, what should be our course of action?

I see the following options:
1. Suppress the error until #2411 gets released
2. Obey current rules of ordering, refactor once #2411 gets released
3. Do nothing, wait until #2411 gets released